manifests/system_table.pp

```diff
@@ -57,22 +60,24 @@
   Optional[String]                         $custom_content = undef
 ) {
 
-  include '::incron'
+  include 'incron'
 
   $_ensure = $enable ? { true => 'present', default => 'absent' }
 
   if $custom_content {
     incron_system_table { $name:
       ensure  => $_ensure,
-      content => $custom_content
+      content => $custom_content,
+      notify  => Class['incron::service']
```

Comment:
Why is this needed? Per the incron man page:
>>If a table (incrontab) is changed incrond reacts immediately and reloads the table
 >> table. Currently running child processes (commands) are not affected.

Reply:
@lnemsick-simp In testing, this did not always appear to occur.